pkg/cluster/components/components.go

```diff
@@ -83,3 +83,10 @@ func getCACertsExtraVolumes() []corev1.Volume {
 	}
 	return volumes
 }
+
+func appendExtraLabels(p *corev1.Pod, labels map[string]string) (err error) {
```

Comment:
style nit, but can you move this up to be with the other append function in the file?